### PR TITLE
Include information for Cassandra-as-a-Service

### PR DESCRIPTION
**Pull request by jorgebay**
I've added a section in the docs for the configuration settings required to use Cassandra as a service.

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @jorgebay,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[https://www.lightbend.com/contribute/cla](https://www.lightbend.com/contribute/cla)

File: docs/src/main/paradox/configuration.md [hunk 1]
```diff
@@ -49,3 +49,27 @@ Alternatively, Akka Discovery can be used for finding the Cassandra server conta
...
+    session-keyspace = my_keyspace
+    cloud {
+      secure-connect-bundle = /path/to/secure-connect-database_name.zip
+    }
+  }
+  advanced {
+    auth-provider {
+      class = PlainTextAuthProvider
+      username = user_name 
+      password = p@ssword1
```
Comment (patriknw) [inline docs/src/main/paradox/configuration.md:71]: Looks like a fake password but maybe make it more clear with
```
username = <user name>
password = <password>
```

File: docs/src/main/paradox/configuration.md [hunk 2]
```diff
@@ -49,3 +49,27 @@ Alternatively, Akka Discovery can be used for finding the Cassandra server conta
 in the @extref:[Alpakka Cassandra documentation](alpakka:cassandra.html#using-akka-discovery).
 
 Without any configuration it will use `localhost:9042` as default.
+
+## Using Cassandra-as-a-Service
```
Comment (patriknw) [inline docs/src/main/paradox/configuration.md:53]: we have separate pages for Scylla and CosmosDB, like https://doc.akka.io/docs/akka-persistence-cassandra/current/cosmosdb.html

I think DataStax Astra should be mentioned in same way.

We should probably also structure it with a separate top level index page for these different Cassandra implementations. I can fix this part afterwards.

File: docs/src/main/paradox/configuration.md (hunk 1 above)
Comment (jorgebay) [reply docs/src/main/paradox/configuration.md]: Makes sense, I've changed it.

File: docs/src/main/paradox/configuration.md (hunk 2 above)
Comment (jorgebay) [reply docs/src/main/paradox/configuration.md:53]: ok, let me know if I should change it.

Comment (patriknw) [reply docs/src/main/paradox/configuration.md:53]: yes, please create a separate page, such as the one for CosmosDB and include in the top level index.md. I can do the adjustments to the structure later.

**Comment by jorgebay** [comment]
I've created a dedicated page for it and include it in the `index.md` file.

**Pull request merged**